components/string_view/string_view.h

```diff
@@ -0,0 +1,40 @@
+#ifndef DATADOG_PHP_STRING_VIEW_H
```

Comment:
should we use the same approach used by @SammyK in https://github.com/DataDog/dd-trace-php/pull/1143/files#diff-b291ab1c1e6a83590ba90d3ee0dbe0167d0fe987786f719eed763ab82434f034R1

TLDR, adding headers to `./datadog` and access them via `#include <datadog/header_file.h>`?

Which one you prefer?